src/nwbinspector/nwbinspector.py

```diff
@@ -150,13 +147,19 @@ def configure_checks(
         checks_out = []
         ignore = ignore or []
         for check in checks:
-            mapped_check = copy_check(check)
+            mapped_check = copy_check(check=check)
             for importance_name, func_names in config.items():
                 if check.__name__ in func_names:
                     if importance_name == "SKIP":
                         ignore.append(check.__name__)
                         continue
                     mapped_check.importance = Importance[importance_name]
+                    # Output wrappers are apparently parsed at time of wrapping not of time of output return...
+                    # Attempting to re-wrap the copied function if the importance level is being adjusted...
+                    # From https://github.com/NeurodataWithoutBorders/nwbinspector/issues/302
+                    # new_check_wrapper = _copy_function(function=mapped_check.__wrapped__)
+                    # new_check_wrapper.importance = Importance[importance_name]
+                    # mapped_check.__wrapped__ = new_check_wrapper
```

Comment:
Leaving this in place for the follow-up where I'll either get it working or remove it in lieu of something simpler